clang/runtime/dpct-rt/include/dpct/bindless_images.hpp

```diff
@@ -949,9 +949,9 @@ dpct_memcpy(sycl::ext::oneapi::experimental::image_mem_handle src,
       sycl::range<3>(w_offset_src / ele_size, h_offset_src, 0);
   const auto dest_offset = sycl::range<3>(0, 0, 0);
   const auto dest_extend = sycl::range<3>(p / ele_size, 0, 0);
-  const auto copy_extend = sycl::range<3>(w / ele_size, h, 0);
+  const auto copy_extend = sycl::range<3>(w / ele_size, h, 1);
   return q.ext_oneapi_copy(src, src_offset, desc_src, dest, dest_offset,
```

Comment:
If the API that coping to the device is different from copying to the host, we should judge the pointer and then choose the right way. Or add a parameter to show the difference.